example.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+from pyroscope import agent
+from multiprocessing import Process
+from threading import Thread
+from time import sleep
+import os
+import signal
+import threading
+
+
+def work(n):
+    i = 0
+    while i < n:
+        i += 1
+
+
+def fast_function():
+    while True:
+        work(25000)
+
+
+def slow_function():
+    while True:
+        work(50000)
+
+
+def killer(p, timeout):
+    sleep(timeout)
+    print("Terminating pid: ", p.pid)
+    os.kill(p.pid, signal.SIGTERM)
+
+
+def start_workers():
+    pr = []
+    pr.append(Process(target=fast_function))
+    pr.append(Process(target=slow_function))
```

Comment:
This probably looks more pythonic:
```suggestion
    pr = [Process(target=fast_function),
          Process(target=slow_function)]
```